nomenclature/definitions/variable/technology/electricity-grid.yaml

```diff
@@ -1,25 +1,90 @@
 # List of variables related to the electricity grid
+# description of variables related to electricity demand
+Value of Lost Load|Electricity:
+   description: Cost of demand-not-served. Cost of load curtailment. Value of Lost Load (VoLL)
+   unit: €/MWh
 
-Network|Electricity|Demand|Reserve|Frequency Containment:
-   description: demand in frequency containment reserve (FCR)            
+Demand not served|Electricity:
+   description: Demand-not-served.
+   unit: MWh
+
+Reserve|Electricity|Requirement|Frequency Containment:
+   description: Demand in frequency containment reserve (FCR)
    unit: MW
-   
-Network|Electricity|Demand|Reserve|Automatic Frequency Restoration:
-   description: demand in automatic frequency restoration reserve (AFRR)             
+
+Reserve|Electricity|Requirement|Automatic Frequency Restoration:
+   description: Demand in automatic frequency restoration reserve (AFRR)
    unit: MW
-   
-Network|Electricity|Demand|Inertia:
-   description: demand in inertia            
+
+Network|Electricity|Requirement|Inertia:
```

Comment:
Is this really a characteristic of the network? Would "Reserve" be a more appropriate category?

Reply:
The location of "Network|Electricity|Requirement|Inertia" is appropriate in the YAML file, I think. I'm not sure about using "Reserve" instead of "Requirement." Are you agree, @sandrinecharousset?

Reply:
@danielhuppmann @erikfilias I think inertia can be seen as a kind of system service such as the reserves; nevertheless it is not a reserve (the unit is not MWh but seconds ); I think it is stongly linked to the network but it is quite difficult to define a category.  I would not use 'reserve', but I am not sure daniel was thinking of replacing requirement by reserve, but network by reserve? In any case, I would not use reserve there

Reply:
Just a quick completement: the term requirement was here to differentiate the needed inertia from the available inertia which is defined in the 'operation' file.

Reply:
@danielhuppmann @erikfilias , in the end, after having a look at the names of variable i would recommend to use network as the category for all 'system services' variables (inertia and reserves), both for requirements and for 'operation'

Reply:
Ok, sounds good to me.